packages/suite/src/components/wallet/Fees/CustomFee/CustomFee.tsx

```diff
@@ -153,6 +155,16 @@ export const CustomFee = <TFieldValues extends FormState>({
                     </Row>
                 </Column>
             )}
+            {cachedBytes && (
```

Comment:
Nit: We have a preference in our codebase to be explicit in conditions.
I don't think send form can ever generate cachedBytes = 0 so practically it wouldn't matter, so here just consistency :slightly_smiling_face: 
```suggestion
            {cachedBytes !== undefined && (
```

Reply:
if we want to be explicit in conditions, it should be `cachedBytes !== undefined || cachedBytes !== false` since the output of `const cachedBytes = transactionInfo && transactionInfo.type !== 'error' && transactionInfo.bytes;` is `number | false | undefined`